### PR TITLE
docs(aws sns): add AWS SNS OOTB connector doc

### PR DESCRIPTION
**Pull request by mbiciin**
## What is the purpose of the change

Add a new piece of documentation for a new AWS SNS connector.

## Are there related marketing activities

N/A

## When should this change go live?

November 15

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/aws-sns.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+To use the **Amazon SNS Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide on using Connectors](../use-connectors.md) to learn more.
+
+## Make your Amazon SNS Connector for sending messages executable
+
+![AWS SNS Filled](../img/connectors-aws-sns-filled.png)
+
+To make your Amazon SNS Connector for sending messages executable, take the following steps:
+
+1. Set the relevant IAM key and secret pair in the **Authentication** section. For example, `secrets.MY_AWS_ACCESS_KEY`. The value can be plain text, but this is not recommended due to security concerns.
+2. In the **Topic Properties** section, set the topicARN of your SNS topic as well as its region.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/aws-sns.md:26]:
```suggestion
2. In the **Topic Properties** section, set the topic ARN of your SNS topic as well as its region.
```

**Review by igpetrov — APPROVED** [review]
Looks great. Thank you.

**Review by igpetrov — DISMISSED** [review]

File: docs/components/connectors/out-of-the-box-connectors/aws-sns.md (hunk 1 above)
Comment (mbiciin) [reply docs/components/connectors/out-of-the-box-connectors/aws-sns.md]: Done! Thank you! :)

**Review by MoAtCamunda — DISMISSED** [review]
Great documentation - no changes from my side :)

**Comment by tmetzke** [comment]
Should we add this to `8.0` and `8.1` docs as well? Technically, you can simply use it there as well.

**Comment by mbiciin** [comment]
Hi, @tmetzke 
How do I do this? Thank you!

**Comment by tmetzke** [comment]
> Hi, @tmetzke How do I do this? Thank you!

There are folders `versioned_docs/version-8.0` and `versioned_docs/version-8.1` in this repository. From there on, it's mostly the same as for `docs`. You add your changes there as well.

For the sidebars, there are also `versioned_sidebars` where you can add your Connector accordingly

Hope that helps.

**Comment by tmetzke** [comment]
> > 
> 
> > Should we add this to `8.0` and `8.1` docs as well? Technically, you can simply use it there as well.
> 
> @tmetzke Do I need to do this and for rabbitMQ , right? #1449

Yes, that would be nice.

**Comment by christinaausley** [comment]
@mbiciin Happy to provide a technical review here as well if you'd like to add me 👍

**Review by christinaausley — APPROVED** [review]
I made a few technical adjustments in my latest commit -- let me know if you have any questions.

**Comment by mbiciin** [comment]
Thank you, @christinaausley! :)

**Comment by tmetzke** [comment]
@mbiciin, can you please also add SNS to `docs/self-managed/connectors-deployment/install-and-start.md` (plus backport this to `8.1` and `8.0`)?

**Comment by mbiciin** [comment]
Hi, @tmetzke 
I updated the documentation as you suggested. Thank you!

**Comment by igpetrov** [comment]
Hi @mbiciin,

Please see failing action `build-docs`.

**Review by igpetrov — DISMISSED** [review]

**Review by igpetrov — DISMISSED** [review]
LGTM, thank you.

**Review by christinaausley — DISMISSED** [review]

**Review by igpetrov — APPROVED** [review]

**Comment by igpetrov** [comment]
Hi @christinaausley,

This one is ready to be merged. Can you do this please as we don't have permissions?

**Review by igpetrov — APPROVED** [review]
LGTM.

FYI @mbiciin, added blurring on screenshots to hide internal AWS account.

**Pull request merged**